src/components/Utils/BannerSubNav.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import styles from "./bannersubnav.module.css";
+
+const BannerSubNav = ({ banner }) => {
+  return (
+    <div>
+      <div className={styles.bgImg}>
+        <img src={banner} alt='' />
+      </div>
+      <div className={styles.navwrapper}>
+        <div className={styles.nav}>
+          <div className={`strikThroghtBtn ${styles.content}`}>
+            <NavLink
+              to='/videos'
+              style={({ isActive }) => {
+                return {
+                  textDecoration: isActive ? "line-through" : "none",
+                };
+              }}>
+              <span>All</span>
+            </NavLink>
+            <NavLink
+              to='/likedvideos'
+              style={({ isActive }) => {
```

Comment:
I think you could avoid writing `inline styles` and toggles classes to do the same.